arcgis-ios-sdk-samples/Features/Time based query/README.md

```diff
@@ -18,8 +18,8 @@ Run the sample, and a subset of records will be displayed on the map. Pan and zo
 2. The `featureRequestMode` of the `AGSServiceFeatureTable` is set to `manualCache`, so that the developer can control how and when the feature table is populated with data.
 3. An `AGSFeatureLayer` is created by passing in the instance of the `AGSServiceFeatureTable`.
 4. An `AGSTimeExtent` object is created by specifying start and end date/time objects.
-5. Apply the `AGSTimeExtent` to an `AGSQueryParmaters` object.
-6. Pass the `AGSQueryParmaters` into `AGSServiceFeatureTable.populateFromService(with:clearCache:outFields:completion:)`.
+5. 5. Apply the `AGSTimeExtent` to an `AGSQueryParameters` object.
```

Comment:
Is there supposed to be an extra `5.` here?

Reply:
🤮 Copy-paste fat-finger slipped in. Fixed.